### PR TITLE
**Breaking:** Added Expand icon

### PR DESCRIPTION
**Pull request by nathanedavis1**
Expand.svg added to icon folder

# Summary

Added Expand.svg icon to icon folder for screen as part of ticket EX-58

# Related issue

# To be tested

Me
- [ ] No error or warning in the console on `localhost:6060`

Tester 1

- [ ] Things look good on the demo.
  

Tester 2

- [ ] Things look good on the demo.

### CODE REVIEW

**Comment by ImogenF** [comment]
I'm not familiar with the new method of adding icons, but it's not showing up in the docs...

**Comment by nathanedavis1** [comment]
@ImogenF Yep, that's my bad. I accidentally added it in a folder. Fabien is on the case :)

File: scripts/build-icons.ts [hunk 1]
```diff
@@ -92,28 +92,33 @@ export const buildIcons = (iconPath?: string) =>
...
+          )
+
+          // Add a preview in the documentation
+          const preview = base64Img.base64Sync(join(inputFolder, fileName))
+          const previewImage = `![${name}Icon](${preview}) `
+          writeFileSync(join(outputFolder, `Icon.${name}.tsx`), output.replace("{{previewImage}}", previewImage))
+          progressBar.tick()
+        } catch (e) {
+          console.error(`\nError: ${name}Icon can't be generated!`)
+          console.error(e)
```
Comment (stereobooster) [inline scripts/build-icons.ts]: `process.exit(1)`?

Comment (TejasQ) [reply scripts/build-icons.ts]: 1 million thumbs up

**Pull request merged**